mysql-test/suite/rocksdb/t/ddse_upgrade_row_lock.test

```diff
@@ -0,0 +1,33 @@
+--source include/have_rocksdb.inc
+--source include/have_innodb_ddse.inc
+
+
+--let $MYSQLD_DATADIR= `select @@datadir`
+
+--let $shutdown_server_timeout= 10
```

Comment:
Why?

Reply:
Removed.. previously thinking is that maybe it is more reliable to shutdown instance and copy data. but looks like it is okay for MTR to copy while instance is running.